.github/ISSUE_TEMPLATE/Standard Issue.md

```diff
@@ -0,0 +1,51 @@
+## Story
+**As a user of <STDcheck, Healthlabs, UTITreatment and/or Manage>**, I need <some functionality expressed as the user of the app would express the need at a high level>.
+
+**I need <a deeper description of the need if necessary (most of the time it will be)>.
+
+**so that <why do you need this (short description of the business value>.
+
+## Business Value  
+
+< Complete description of the business value - why do we need to make this change - for the requested item. It is useful to have this even for bugs that we feel “must” be fixed because it helps prioritization. >
+
+## Acceptance Criteria
+< a numbered list of behaviors that must be present in order to satisfy the required functionality described above. >
+
+1. 
+2. 
+ 
+## Notes
+< bulleted list of special notes or suggestions to the development team on the implementation of the changes. >
+
+*  
+
+
+## Definition of Ready
+
+- [ ] The Issue maintains standard format for Issues and meets INVEST
+- [ ] Issue Defined and written appropriately
+- [ ] Acceptance Criteria Defined
+- [ ] The person who will Approve story identified
+- [ ] Dependencies Identified
+- [ ] Issue size estimated by Dev Team
+- [ ] Proposed solution accepted
+
+## Definition of Done
+
+- [ ] Issue accepted by PO
+- [ ] Issue is scored
+- [ ] All features implemented
+- [ ] All features not yet implemented or inactive are hidden from the user
+- [ ] Peer Reviews complete	
+- [ ] Unit and regression tests are Passed
+- [ ] Manual Testing Complete and issues resolved
+- [ ] Automated tests created and run successfully
+- [ ] All blockers addressed
+- [ ] Acceptance criteria met
+- [ ] Deployed and verified in Production
+- [ ] Ready for PO Approval
+
+## Proposed Solution  
```

Comment:
Can we add a section called:
```
## Subject Domain Experts
```

So the person work on the issue knows who should consult

Reply:
Good suggestion. Do we have a published list of SMEs anywhere so the creator can put them in?